### PR TITLE
Replaces style attribute with class for spinner

### PR DESCRIPTION
**Pull request by Kristyn-Mimura**
# Ticket Link

[Ticket](https://uhawaii.atlassian.net/browse/GROUPINGS-1427)

# List of squashed commits
- Updates spacing

# Test Checklist

- [X] Unit Tests Passed:
- [x] Jasmine Tests Passed:
- [X] General Visual Inspection:

### CODE REVIEW

**Review by mhodgesatuh — APPROVED** [review]
Nice cleanup

**Comment by mhodgesatuh** [comment]
src/main/resources/static/scss/components/_spinner.scss
Minor icon MINOR
Code Style
Unexpected missing end-of-source newline (no-missing-end-of-source-newline)
}

Looks like one bit of cleanup to take care of.

**Comment by Kristyn-Mimura** [comment]
> src/main/resources/static/scss/components/_spinner.scss Minor icon MINOR Code Style Unexpected missing end-of-source newline (no-missing-end-of-source-newline) }
> 
> Looks like one bit of cleanup to take care of.

I added a newline, squashed the commits then pushed to origin but the newline didn't show up, I'm trying to figure out what went wrong

**Comment by mhodgesatuh** [comment]
Hopefully your IDE is not configured to remove the end-of-source newline.

**Comment by JorWo** [comment]
Maybe try add two newlines at the end of _spinner.scss to fix the codacy issue.

**Comment by Kristyn-Mimura** [comment]
> Maybe try add two newlines at the end of _spinner.scss to fix the codacy issue.

Thanks! That worked!

**Comment by JorWo** [comment]
A recent merge just created a merge conflict.

File: src/main/resources/templates/admin.html [hunk 1]
```diff
@@ -32,8 +32,8 @@ <h1 class="text-center text-md-left">UH Groupings Administration</h1>
     <div class="container">
         <div id="overlay" ng-show="loading">
             <div class="d-flex justify-content-center">
-                <div class="spinner-border text-light" style="width: 7rem; height: 7rem; margin-top: 17em;" ng-show="loading"></div>
-            </div>
+            <div class="spinner-border text-light spinner-style" ng-show="loading"></div>
+                </div>
```
Comment (JorWo) [inline src/main/resources/templates/admin.html:36]: Could you fix the spacing here? After that, everything in this PR is good to go.

**Comment by JorWo** [comment]
Sorry about that, there's another merge conflict.

**Review by JorWo — APPROVED** [review]

**Pull request merged**